ush/python/pygw/src/pygw/jinja.py

```diff
@@ -1,44 +1,191 @@
 import io
-import os
 import sys
 import jinja2
+from markupsafe import Markup
 from pathlib import Path
 
+from .timetools import strftime, to_YMDH, to_YMD, to_fv3time, to_isotime
+
+
+@jinja2.pass_eval_context
+class SilentUndefined(jinja2.Undefined):
```

Comment:
if it is internal only, can you make the class `_SilentUndefined`? Or is that not PEP8 kosher?

Reply:
It is internal only.
I can define a `__all__` and not include in it to make it proper.
Thanks for the reminder